### PR TITLE
User roles

### PR DESCRIPTION
**Pull request by trombomart**

### CODE REVIEW

File: src/main/java/nl/tudelft/thefirstorder/security/AuthoritiesConstants.java [hunk 1]
```diff
@@ -11,6 +11,12 @@
 
     public static final String ANONYMOUS = "ROLE_ANONYMOUS";
 
+    public static final String DIRECTOR = "ROLE_DIRECTOR";
+
```
Comment (rubenwiersma) [inline src/main/java/nl/tudelft/thefirstorder/security/AuthoritiesConstants.java]: Hier kunnen witregels weg.

**Comment by rubenwiersma** [comment]
Naast witregels niks kunnen ontdekken

**Comment by ChrisBerg** [comment]
Looks nice, keep up the good work!

**Comment by hungnguyen94** [comment]
Looks nice. Have you also created users with these authority? 
That way, we can test it for ourselves.

**Comment by trombomart** [comment]
@hungnguyen94 No there aren't users with these authorities but you can test this with the differences between user and admin.

**Comment by cadw** [comment]
Looks fine to me, I'll merge it.

**Pull request merged**